packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -89,11 +61,6 @@ export async function setActiveAccount(account: Account): Promise<void> {
       auth: true,
     });
 
-    app.roles.getAllRolesInCommunity({ community }).forEach((r) => {
-      r.is_user_default = false;
-    });
-    role.is_user_default = true;
```

Comment:
i remember that this piece of code was necessary in the situation when we wanted to persist the default address selected when you reload the page/open the app again. If this still works, then fine

Reply:
The /setDefaultRole call should be enough -- updating the roles controller should affect the *current* active address, and not the active address on reload. That said, I will test this functionality now.

Reply:
On testing, the active address persists both locally and across refreshes, so we should be OK here.

Reply:
thanks for checking